packages/vite-plugin-svelte/src/utils/optimizer-plugins.js

```diff
@@ -1,17 +1,33 @@
 import { readFileSync } from 'node:fs';
 import * as svelte from 'svelte/compiler';
 import { log } from './log.js';
-import { toESBuildError } from './error.js';
+import { toESBuildError, toRollupError } from './error.js';
 import { safeBase64Hash } from './hash.js';
 import { normalize } from './id.js';
 
 /**
  * @typedef {NonNullable<import('vite').DepOptimizationOptions['esbuildOptions']>} EsbuildOptions
  * @typedef {NonNullable<EsbuildOptions['plugins']>[number]} EsbuildPlugin
  */
-
-export const facadeEsbuildSveltePluginName = 'vite-plugin-svelte:facade';
-export const facadeEsbuildSvelteModulePluginName = 'vite-plugin-svelte-module:facade';
+/*
+ * @typedef {NonNullable<import('vite').DepOptimizationOptions['rollupOptions']>} RollupOptions
+ * @typedef {NonNullable<RollupOptions['plugins']>[number]} RolldownPlugin
```

Comment:
even if i were to change `vite` in line 13 to `rolldown-vite` rollupOptions.plugins would not be detected here.

so below is a shim that works for our use.

Reply:
thanks!! used ` * @typedef {import('vite').Rollup.Plugin} RollupPlugin`